saleor/graphql/discount/tests/mutations/test_voucher_code_bulk_delete.py

```diff
@@ -0,0 +1,80 @@
+from unittest import mock
+
+import graphene
+
+from ....tests.utils import get_graphql_content
+
+VOUCHER_CODE_BULK_DELETE_MUTATION = """
+    mutation voucherCodeBulkDelete($ids: [ID!]!) {
+        voucherCodeBulkDelete(ids: $ids) {
+            count
+            errors {
+                field
+                message
+            }
+        }
+    }
+"""
+
+
+def test_delete_voucher_codes(staff_api_client, voucher, permission_manage_discounts):
+    # given
+    voucher.codes.create(code="voucher-1")
+    voucher.codes.create(code="voucher-2")
+    assert voucher.codes.count() == 3
```

Comment:
You could do
```
vouchers = voucher.codes.all()
assert len(vouchers) == 3
```
And then in variables you wouldn't have to query for all codes again, saving one db query 👯